cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1282,6 +1292,251 @@ def _check_column_availability(self):
                 if "add_labels" in index_def:
                     self._check_single_column_availability(component, index_def["add_labels"])
 
+    def _check_spatial(self):
+        """
+        Validate spatial-related values of the AnnData object. Validation is not defined in schema definition yaml.
+        Errors are added to self.errors.
+
+        :rtype none
+        """
+
+        # Exit if uns is not specified. Error is reported in core validate functionality.
+        uns_component = getattr_anndata(self.adata, "uns")
+        if uns_component is None:
+            return
+
+        # uns spatial validation is dependent on obs.assay_ontology_term_id; exit if not specified. Error is
+        # reported in core validate functionality.
+        obs_component = getattr_anndata(self.adata, "obs")
+        if obs_component is None or "assay_ontology_term_id" not in obs_component:
+            return
+
+        # spatial is forbidden if assay it not a supported spatial assay.
+        uns_spatial_specified = "spatial" in self.adata.uns
```

Comment:
nit: if you can replace uns_spatial_specified with `uns_spatial = self.adata.uns.get("spatial")` since it returns None if uns_spatial is not specified. one less var to maintain